### ISSUE TITLE
Write some tests

### ISSUE THREAD

**Issue by peterbe** [peterbe/minimalcss#4]
There are currently exactly 0 tests and no infra for running tests. 
It would be nice to have some and be able to install a CI (Travis or Circle).

**Comment by stereobooster** [comment]
I suppose we need to stop development and add some tests before we can move further. Cost of manual testing starts to outweigh time to implement some tests

**Comment by stereobooster** [comment]
What else do we need?

- [ ] `@keyframe`
- [ ] `@fontface`
- [ ] `@keyframe` used in internal styles
- [ ] `@fontface` used in internal styles
- [ ] `:hover`
- [ ] `@media`
- [ ] `@media` with print

**Comment by peterbe** [comment]
All of everything? 

One nice test to have would be to check that CSS that is only relevant *before* the XHRs finish, is still there. E.g. That `.loaded` is still in the output CSS string even though [XHR callbacks delete its DOM](https://github.com/peterbe/minimalcss/blob/master/examples/stages.html#L25). 

Perhaps tests for the cli too. Perhaps we can mock it so it every actually calls `minimalcss.minimize` but calls a mock function that collects and analyzes the options sent to it.

**Comment by stereobooster** [comment]
We have #49 for CLI tests, and separate tickets for everything else. Solved by #104
